scripts/core/CONTRIB.rst

```diff
@@ -56,6 +56,15 @@ Adapter Change Process
     `oneapi-src/unified-runtime`_ project targeting the `main
     <https://github.com/oneapi-src/unified-runtime/tree/main>`_ branch.
 
+    .. note::
+        Historically, convention on the project has been to add one or more
+        ``[COMPONENT]`` prefixes to the pull request title to signify which
+        components are changed within. This is no longer necessary as pull
+        requests are now automatically labeled by the `Pull Request Labeler
+        <https://github.com/oneapi-src/unified-runtime/blob/main/.github/labeler.yml>`_
+        GitHub Action. This approach is less error prone and much easier to
+        filter on the GitHub pull requests tab.
```

Comment:
Perhaps add something about how convention dictates that titles are only 50 characters long, and without the tags this is now available for use?

Reply:
Yeah. That's a good change. :thum